Provide means to delete all Solr documents

skng5 opened the pull request:
- Issue #23
- Creates delete_all method and means to activate via Rake and command
  line
- Provides separate commit task to facilitate seamless purge and
  reharvest of libguides
- Updates Ruby version to 2.4.1

nomadicoder committed:
Provide means to delete all Solr documents

- Issue #23
- Creates delete_all method and means to activate via Rake and command
  line
- Provides separate commit task to facilitate seamless purge and
  reharvest of libguides
- Updates Ruby version to 2.4.1

bibliotechy commented on lib/guiding_light/harvest.rb:
```diff
@@ -80,6 +80,46 @@ def self.harvest_all
+    #
+    solr = RSolr.connect url: config.solr_url
+
+    libguide_sites = GuidingLight::Request.get_guides(config.api_url, config.site_id, config.api_key)
```
Does this need to be here still?

nomadicoder committed:
Remove development leftover code

- Removed code from develoment process
- Moved all solr code into rescue blocks
